docs/en/sql-reference/statements/system.md

```diff
@@ -119,6 +119,18 @@ The compiled expression cache is enabled/disabled with the query/user/profile-le
 
 Clears the [query cache](../../operations/query-cache.md).
 
+## DROP SCHEMA FORMAT CACHE {#system-drop-schema-format}
```

Comment:
Just curious why you chose order `SCHEMA FORMAT CACHE` and not `FORMAT SCHEMA CACHE`. The second variant sound more natural for me.

Reply:
To be honest, I wouldn't say I like any of them (there are too many words in the query and it's hard to remember). But comparing them again, I can agree that the second variant might be more natural. 

So, yep, I changed it.